pkg/autoscaler/autoscaler.go

```diff
@@ -338,7 +325,7 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		Expect(clusterExpansionSize).To(BeNumerically(">", 1))
 
 		// The total size of our cluster is
-		// len(existingMachineSets) + clusterExpansionSize. We
+		// len(nodes) + clusterExpansionSize. We
 		// cap that to $max-1 because we want to test that the
```

Comment:
cleaning up the comments to show the actual behavior. we aren't using `existingMachineSets` and instead query the number of nodes just before the following assignment.